moleculegen/data.py

```diff
@@ -266,7 +266,7 @@ def get_valid_lengths(sample: np.ndarray) -> np.ndarray:
             # FIXME Iterating over matrix is somewhat brute and ineffective.
```

Comment:
We want to use an `apply` method instead of `iter`, but the current version of deepnumpy has no `apply`s.